rootfs/builder/build.sh

```diff
@@ -25,10 +25,10 @@ if ! [[ -z "${TAR_URL}" ]]; then
 			tar -xzf /tmp/slug.tgz -C /app/
 			unset TAR_URL
 		fi
+	else
+		curl -s "$TAR_URL" | tar -xzC /app/
+		unset TAR_URL
 	fi
-else
```

Comment:
i think we still want this logic here. if $TAR_URL is present but the secret is not, we should still pull the tarball.